### PR TITLE
migrate to copier template

### PR DESCRIPTION
**Pull request by pavelzw**
migrate to copier template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,18 @@
-# shellcheck pre-commit hook
+# shellcheck mirror
 
-Hook for shellcheck with conda as a language.
+Mirror of shellcheck for pre-commit with conda as a language.
```
Comment (0xbe7a) [inline README.md:3]: What is this hook mirroring?

**Review by 0xbe7a — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pavelzw) [reply README.md:3]: https://github.com/koalaman/shellcheck-precommit when you broaden the term "mirroring"

Comment (pavelzw) [reply README.md:3]: Also, this repo's name is pre-commit-_mirrors_-shellcheck

Comment (pavelzw) [reply README.md:3]: Fixed in the next copier release

**Pull request merged**